File: app/src/main/java/com/worlyep/studyfork/model/Song.java
package com.worlyep.studyfork.model;

import com.worlyep.studyfork.constant.KaraokeType;

public class Song {
    private String singer;
    private KaraokeType karaokeTypeType;
    private int code;
    private boolean isFavorites;
    private String title;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getSinger() {
        return singer;
    }

    public void setSinger(String singer) {
        this.singer = singer;
    }

    public KaraokeType getKaraokeTypeType() {
        return karaokeTypeType;
    }

    public void setKaraokeTypeType(KaraokeType karaokeTypeType) {
        this.karaokeTypeType = karaokeTypeType;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public boolean isFavorites() {
        return isFavorites;
    }

    public void setFavorites(boolean favorites) {
        isFavorites = favorites;
    }
}

File: app/src/main/java/com/worlyep/studyfork/model/Song.kt
package com.worlyep.studyfork.model

import com.google.gson.annotations.SerializedName
import com.worlyep.studyfork.constant.KaraokeType

class Song {
    @SerializedName("singer") var singer: String? = null
    @SerializedName("brand") var karaokeTypeType: String? = null
    @SerializedName("no") var code: String? = null
    @SerializedName("title") var title: String? = null
    var isFavorites: Boolean = false
}
